src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.html

```diff
@@ -0,0 +1,549 @@
+                          *ngIf="initiator.showError('client_iqn', formDir, 'pattern')"
+                          i18n>IQN has wrong patter.</span>
+                  </div>
+                </div>
+
+                <ng-container formGroupName="auth">
+                  <!-- Initiator: User -->
+                  <div class="form-group"
+                       [ngClass]="{'has-error': initiator.showError('user', formDir)}">
+                    <label class="control-label col-sm-3"
+                           for="user"
+                           i18n>User</label>
+                    <div class="col-sm-9">
+                      <input id="user"
+                             class="form-control"
+                             formControlName="user"
+                             type="text">
+                      <span class="help-block"
+                            *ngIf="initiator.showError('user', formDir, 'required')"
+                            i18n>This field is required.</span>
+
+                      <span class="help-block"
+                            *ngIf="initiator.showError('user', formDir, 'pattern')"
+                            i18n>Usernames must have a length of 8 to 64 characters and
+                        can only contain letters, '.', '@', '-', '_' or ':'.</span>
+                    </div>
+                  </div>
+
+                  <!-- Initiator: Password -->
+                  <div class="form-group"
+                       [ngClass]="{'has-error': initiator.showError('password', formDir)}">
+                    <label class="control-label col-sm-3"
+                           for="password"
+                           i18n>Password</label>
+                    <div class="col-sm-9">
+                      <div class="input-group">
+                        <input id="password"
+                               class="form-control"
+                               formControlName="password"
+                               type="password">
+
+                        <span class="input-group-btn">
+                          <button type="button"
+                                  class="btn btn-default"
+                                  cdPasswordButton="password">
+                          </button>
+                          <button type="button"
+                                  class="btn btn-default"
+                                  cdCopy2ClipboardButton="password">
+                          </button>
+                        </span>
+                      </div>
+                      <span class="help-block"
+                            *ngIf="initiator.showError('password', formDir, 'required')"
+                            i18n>This field is required.</span>
+
+                      <span class="help-block"
+                            *ngIf="initiator.showError('password', formDir, 'pattern')"
+                            i18n>Passwords must have a length of 12-16 and
```

Comment:
```suggestion
                            i18n>Passwords must have a length of 12 to 16 characters and
```
See https://github.com/ceph/ceph/pull/25995/files#diff-37b56750ebccce5725fc786fa1762a59R235